SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -28,8 +28,8 @@ bool FSpatialGDKEditor::GenerateSchema(bool bFullScan)
 
 	// Prompt the user to save packages/maps.
 	const bool bPromptUserToSave = true;
-	const bool bSaveMapPackages = true;
-	const bool bSaveContentPackages = true;
+	const bool bSaveMapPackages = false;
+	const bool bSaveContentPackages = false;
```

Comment:
Just curious, why did these get changed?

Reply:
Presumably this is for the issue in UNR-1312. Hopefully https://github.com/spatialos/UnrealGDK/pull/881/files fixes this.

Reply:
In the meantime this change causes crashes when there are dirty assets and schema is being generated in the editor.

Reply:
Oh oops, this was part of my workaround for UNR-1312. No idea how these ended up in here, good catch. Will revert.